upup/pkg/fi/cloudup/dns.go

```diff
@@ -37,7 +37,7 @@ const (
 	// This IP is from TEST-NET-3
 	// https://en.wikipedia.org/wiki/Reserved_IP_addresses
 	PlaceholderIP  = "203.0.113.123"
-	PlaceholderTTL = 10
+	PlaceholderTTL = 60
```

Comment:
Maybe we should add a comment here that DigitalOcean's DNS servers require a certain minimum TTL (IIRC, it's 30)?

Reply:
Is it feasible to set the 60 second TTL if this is DO (I think it is, because it looks like already check the cloudprovider in precreateDNS where this is used)?  This is an optimization that helps prevent backoff, and I'd prefer not to risk slowing bringup down across the board, if we can.  (And then we can have the comment also on `PlaceholderTTLDigitalOcean` :-) )

Reply:
Sure, will do this change so it doesn't affect any other cloud providers.